cpg-analysis/src/test/kotlin/de/fraunhofer/aisec/cpg/analysis/ValueEvaluatorTest.kt

```diff
@@ -770,4 +770,23 @@ class ValueEvaluatorTest {
             assertEquals(2.5f, ValueEvaluator().evaluate(minusminus))
         }
     }
+
+    @Test
+    fun handleConditionalExpression() {
+        with(TestLanguageFrontend()) {
+            val a = newVariableDeclaration("a")
+            a.initializer = newLiteral(1)
+
+            val aRef = newReference("a")
+            aRef.refersTo = a
+            aRef.prevDFG = mutableSetOf(a)
+
+            val comparison = newBinaryOperator("!=")
+            comparison.lhs = aRef
+            comparison.rhs = newLiteral(1)
+
+            val cond = newConditionalExpression(comparison, newLiteral(2), aRef)
```

Comment:
Not sure why this test ever worked because we didn't handle the `!=` operator if I see this correctly

Reply:
This is a new test :)

Reply:
Hmm interesting ^^